src/Router.ts

```diff
@@ -669,6 +831,24 @@ class RouterImpl {
 		// Return the key value pairs.
 		return pathParams;
 	}
+
+	/**
+	 * Get the full path and URL parameters string for a given path, removing the protocol, server, and port information.
+	 * 
+	 * @param source - The URL to inspect.
+	 * 
+	 * @returns The path string.
+	 */
+	private _getPathString(source: string): string {
+
+		const result = source.replace(window.location.origin, '');
+
+		if (!result || result === '') {
+			return '/';
```

Comment:
Are we not making some assumptions here by defaulting to `/`? Based on previous code we did not transform that after the string replace so this might introduce breaking changes potentially. Would it not be prudent to allow configuring this default return as well?

Reply:
Reverted to align with original